pymeasure/instruments/diamondengineering/damsx000.py

```diff
@@ -302,14 +323,19 @@ def steps2degrees(self, steps):
         ys = R * sin(angle_rad) + h * cos(angle_rad)
         ds = self.distance(xs, ys, self.x0, self.y0)
 
-        d1 = steps/self.steps_per_meter + ds
-        x1,y1,x2,y2 = self.get_intersections(d1)
+        d1 = -steps/self.steps_per_meter + ds
```

Comment:
```suggestion
        d1 = steps/self.steps_per_meter + ds
```
To be consistent with changes in `degrees2steps`

Reply:
Solved in a different way by changing the code for angle manipulation